solr/core/src/java/org/apache/solr/handler/component/QueryElevationComponent.java

```diff
@@ -413,17 +414,16 @@ private ElevationProvider loadElevationProvider(SolrCore core) throws Exception
    * @throws RuntimeException If the config does not provide an XML content of the expected format
    *     (either {@link RuntimeException} or {@link org.apache.solr.common.SolrException}).
    */
-  protected ElevationProvider loadElevationProvider(XmlConfigFile config) {
+  protected ElevationProvider loadElevationProvider(Document doc) {
     Map<ElevatingQuery, ElevationBuilder> elevationBuilderMap = new LinkedHashMap<>();
+    NodeList nodes = doc.getDocumentElement().getElementsByTagName("query");
```

Comment:
Check that the document element has the name we expect "elevate"; otherwise maybe the wrong XML file was given and the user would be confused.

Reply:
Thanks for the reminder David. I deleted it by mistake. Pushing changes.